engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -133,9 +133,9 @@ public boolean incrementEndpointServerErrors(UUID endpointId, int maxServerError
             if (endpoint.get().getServerErrors() + 1 > maxServerErrors) {
                 /*
                  * The endpoint exceeded the max server errors allowed from configuration.
-                 * It is therefore disabled and the errors counter is reset.
+                 * It is therefore disabled.
                  */
-                String hql = "UPDATE Endpoint SET serverErrors = 0, enabled = FALSE WHERE id = :id";
+                String hql = "UPDATE Endpoint SET enabled = FALSE WHERE id = :id";
```

Comment:
one question though, now that we don't reset and we still keep trying on failure (because the 3 retries) - maybe this is going to send 3 emails instead of one? (one for the 10 retry, 11 retry and 12 retry)

Reply:
Correct, but there is an easy fix for that and I'll push it right away.